org.mwc.debrief.lite/build.xml

```diff
@@ -151,7 +151,7 @@
 			<fileset dir="../org.mwc.debrief.legacy/libs" includes="*.jar"/>
 			<fileset dir="../org.mwc.cmap.jfreechart/libs" includes="*.jar"/>
 			<fileset dir="../org.mwc.cmap.geotools/libs" includes="*.jar"/>
-			<fileset dir="${basedir}" includes="cmap_legacy.jar"/>
+			<fileset dir="../org.mwc.debrief.legacy" includes="cmap_legacy.jar"/>

```

Comment:
@ayeshare - wouldn't `org.mwc.cmap.legacy` be a tidier source folder to take it from?  That's where the plugin is created/

Reply:
yes, but we are building cmap from debrief.legacy, so the basedir will be debrief.legacy.